WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -297,7 +314,15 @@ protected SendControlViewModel(WalletService walletService, string title)
 
 					var feeStrategy = FeeStrategy.CreateFromFeeRate(FeeRate);
 
-					var intent = new PaymentIntent(address, moneyRequest, label);
+					var activeDestinationRequest = new DestinationRequest(address, moneyRequest, label);
+					var requests = new List<DestinationRequest>() { activeDestinationRequest };
+
+					if (customChangeAddress is { })
```

Comment:
The same as before. Guarantee to not use custom change if it is turned off in the config.

Reply:
wontfix